src/foam/nanos/auth/FilterBySpidService.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+   package: 'foam.nanos.auth',
+   name: 'FilterBySpidService',
+   extends: 'foam.nanos.auth.ProxyUserLocatorService',
+   flags: ['java'],
+
+  documentation: `Restrict login to the url that matches the spid of the user.`,
```

Comment:
i dont think this service restricts anything, this locator just filters out usrs

Reply:
thank you, I changed this.